addons/bestja_volunteer/models.py

```diff
@@ -126,48 +135,63 @@ def sync_group(self, group, domain):
         })
 
     def check_password_rules(self, password):
```

Comment:
Since we are changing this method anyway and since it returns a boolean value, I think that a better name would be something like `new_password_is_valid`. Also it may be changed to static method.